lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -925,33 +933,40 @@ findMatchingConcatDimForSameSizeSubTensor(llvm::ArrayRef<size_t> firstDims,
 /// there is a dimension in the input tensors, with which we can meet
 /// two requirements:
 /// 1) Input tensors are concatenate-able along this dimension.
-/// 2) The sub-tensors after this dimension in the input tensors, are
-///    of the same size as the sub-tensor of the input of the original
-///    Concat node after the concatenation dimension.
+/// 2) The trailing/leading dimensions sizes after/before this dimension in the
+///    input tensors, are of the same size as the trailing/leading dimensions of
+///    the input of the original Concat node after/before the concatenation
+///    dimension.
 /// \returns this dimension if found, otherwise -1.
 static int tryToConcatAlongSameSizeSubTensor(llvm::ArrayRef<NodeValue> inputs,
                                              ConcatNode *origConcatN) {
```

Comment:
"origConcatN" stays for "original Concat Node" I guess? Later you use names like "trailingDimsProdOriginalConcatNode", which is inconsistent. 

Either use "ConcatNode" in such cases everywhere, or even better "CN" to make the names a bit shorter as they are getting really long. 

In any case, please pick one way of naming things and use it consistently in the code.

Reply:
changed to originalConcatNode